### PR TITLE
probe power constraint scaling and fft using backward normalization

### PR DESCRIPTION
**Pull request by a4894z**
When using the probe power constraint, because of the way PtyChi does fft scaling, the power the probe gets scaled to is P / N^2 (where the array size of the probe is N x N) instead of P. 

This small fix should account for this; alternatively we can initialize the probe power constraint to be P * N^2 and not use this small fix, but that seems like a clunky way of doing things?

Another way to account for this is to not use the propagate_forward() part of the code, directly compute the probe norm e.g. torch.sum(probe_composed.abs() ** 2), and then rescale the probe and sample using this power correction?

### CODE REVIEW

**Comment by mdw771** [comment]
Sorry it slipped out of my sight..merging it now
